
+++ //depot/CommEngine/patches/WLAN/PATCH_502L07_P1/targets/arch/WL20D1GA.arch
--- //depot/CommEngine/patches/WLAN/PATCH_502L07_P1/targets/arch/WL20D1GA.arch

@@ -0,0 +1,6 @@
+<PUBLISH>
+--- 963138GW.old	2019-11-13 16:15:50.981302630 -0800
++++ 963138GW	2019-11-13 16:17:01.137792078 -0800
+@@ -286,7 +286,7 @@
+-BCM_WLIMPL=
++BCM_WLIMPL=67
